metron-analytics/metron-profiler-client/src/test/java/org/apache/metron/profiler/client/ProfileWriter.java

```diff
@@ -92,8 +92,8 @@ private void write(ProfileMeasurement m, List<Object> groups) {
     byte[] rowKey = rowKeyBuilder.rowKey(m, groups);
     ColumnList cols = columnBuilder.columns(m);
 
-    List<Mutation> mutations = hbaseClient.constructMutationReq(rowKey, cols, Durability.SKIP_WAL);
-    hbaseClient.batchMutate(mutations);
+    hbaseClient.addMutation(rowKey, cols, Durability.SKIP_WAL);
```

Comment:
The `ProfileWriter` is only used for testing.  It writes Profile data that I can then read with the client API.  I have it in under `src/test/...`.